wdyr.ts

```diff
@@ -1,9 +1,8 @@
-// Implements Why Did You Render (WDYR) in Dev
-import lodashGet from 'lodash/get';
+/// <reference types="@welldone-software/why-did-you-render" />
```

Comment:
this done according to TS description in: https://github.com/welldone-software/why-did-you-render?tab=readme-ov-file#setup